Error building APK

**Mobile Device Environment**
Provide a list of operating systems on which this issue is relevant.
- Device: Android - compileSdkVersion 31

**Application Environment**
Provide information about your development environment:
- React Native 0.7 with RN Bluetooth Classic v1.60.0

**Describe the bug**
I'm trying to create an APK of my react-native app running assembleRelease and i'm getting this error at the end of the command:
**"> Task :react-native-bluetooth-classic:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-bluetooth-classic:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:C:\Users\Tecnoandina\.gradle\caches\transforms-3\efc5fbdc5cddabff1e6dbe6ad771c56d\transformed\core-1.7.0\res\values\values.xml:105:5-114:25: AAPT: error: resource android:attr/lStar not found."**


Any suggestion? What must I do? :(

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -11,13 +11,17 @@ buildscript {
 
 apply plugin: 'com.android.library'
 
+def getExtOrDefault(name, fallback) {
+  return rootProject.ext.has(name) ? rootProject.ext.get(name) : fallback
+}
+
 android {
-    compileSdkVersion 29
-    buildToolsVersion "28.0.3"
+    compileSdkVersion getExtOrDefault('compileSdkVersion', 29)
+    buildToolsVersion getExtOrDefault('buildToolsVersion', "28.0.3")
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 29
+        minSdkVersion getExtOrDefault('minSdkVersion', 16)
+        targetSdkVersion getExtOrDefault('targetSdkVersion', 29)
         versionCode 1
         versionName "1.0"
     }

